examples/ct/structures_test.go

```diff
@@ -161,3 +261,20 @@ func setupMockKeyManager(toSign []byte) crypto.KeyManager {
 
 	return mockKeyManager
 }
+
+// Creates a dummy cert chain
+func createCertChain(numCerts int) []ct.ASN1Cert {
```

Comment:
Yes, this is all just a container for bytes that gets pushed to the backend and then retrieved to serve get-entries etc.